pkgs/misc/emulators/openmsx/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, pkgconfig
+, python
+, alsaLib, glew, mesa_noglu, libpng
+, libogg, libtheora, libvorbis
+, SDL, SDL_image, SDL_ttf
+, freetype, tcl, zlib
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "openmsx-${version}";
+  version = "git-2017-11-02";
```

Comment:
Why not 0.14.0? (https://github.com/openMSX/openMSX/releases/tag/RELEASE_0_14_0)

Reply:
I have talked with the openMSX developers in their IRC channel, and after presenting a bit of NixOS to them, they added a simple patch to make it build under Nix.

Do you think using the patch file is better than the hardcoded git?

https://github.com/openMSX/openMSX/commit/eeb74206ae347a3b17e9b99f91f2b4682c5db22c

Reply:
ping @yegortimoshenko